### PR TITLE
[doc]configuration descriptions

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
as title

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+属性名                   | 默认值            | 说明
+------------------------------- | ------------------------ | -----------
+`port`                          | 3699                     | Nebula Graph daemon 监听端口
+`client_idle_timeout_secs`      | 0                        | 关闭 idle 连接前的时长（单位秒）， 0 为无穷大
+`session_idle_timeout_secs`     | 600                      | idle sessions 过期时长（单位秒），0 为无穷大
+`session_reclaim_interval_secs` | 10                       | 超出指定时间则认为超时
+`num_netio_threads`             | 0                        | networking 线程数，0为物理 CPU 核数
+`num_accept_threads`            | 1                        | 接受进入连接的线程数
+`num_worker_threads`            | 0                        | 执行用户查询的线程数
+`reuse_port`                    | true                     | 开启 SO_REUSEPORT 选项
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: graphd default is `false`

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+属性名                   | 默认值            | 说明
+------------------------------- | ------------------------ | -----------
+`port`                          | 3699                     | Nebula Graph daemon 监听端口
+`client_idle_timeout_secs`      | 0                        | 关闭 idle 连接前的时长（单位秒）， 0 为无穷大
+`session_idle_timeout_secs`     | 600                      | idle sessions 过期时长（单位秒），0 为无穷大
+`session_reclaim_interval_secs` | 10                       | 超出指定时间则认为超时
+`num_netio_threads`             | 0                        | networking 线程数，0为物理 CPU 核数
+`num_accept_threads`            | 1                        | 接受进入连接的线程数
+`num_worker_threads`            | 0                        | 执行用户查询的线程数
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: 执行用户请求的线程数，线程数为系统cpu核数

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+`stdout_log_file`               | "graphd-stdout.log"      | stdout 目标文件名
+`stderr_log_file`               | "graphd-stderr.log"      | stderr 目标文件名
+`daemonize`                     | true                     | 作为 daemon 进程运行
+`meta_server_addrs`             | ""                       |  meta server 地址列表，格式为 ip1:port1, ip2:port2, ip3:port3
+
+## Web Service
+
+属性名         | 默认值 | 说明
+---------------------- | ------------- | -----------
+`ws_http_port`         | 11000         |  HTTP 协议监听端口
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: 这里三个服务都改为`ws_http_port`和`ws_ip`， graphd http默认端口号为13000

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+`ws_meta_http_port`    | 11000         |  Meta HTTP 协议监听端口
+`ws_meta_h2_port`      | 11002         |  Meta HTTP/2 协议监听端口
+`ws_storage_http_port` | 12000         | Storage HTTP 协议监听端口
+`ws_storage_h2_port`   | 12002         | Storage HTTP/2 协议监听端口
+
+## Console
+
+属性名            | 默认值 | 说明
+------------------------ | ------------- | -----------
+`addr`                   | "127.0.0.1"   | Nebula daemon IP 地址
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: `Nebula daemon IP 地址` 改为`Graph daemon IP`

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+`port`                   | 0             | Nebula daemon 监听端口
```
Comment (laura-ding) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+# 配置文件说明
+
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: Do you need to add some short description for this reference guide?

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+## Meta Service
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: Please add some short description for tables

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md (hunk 6 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: The parameters are described in the table.

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
+# Configuration Description
+
+## Meta Service
+
+Property Name               | Default Value            | Description
+--------------------------- | ------------------------ | -----------
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: Please keep the same style for the description field. Whether a period is needed for each description?

Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: fixed

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md (hunk 6 above)
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md]: 本文介绍 etc/ 目录下配置文件所对应参数。

**Pull request merged**